common/src/jni/main/cpp/conscrypt/jniutil.cc

```diff
@@ -297,6 +297,7 @@ int throwForAsn1Error(JNIEnv* env, int reason, const char* message,
         case ASN1_R_WRONG_PUBLIC_KEY_TYPE:
             return throwInvalidKeyException(env, message);
             break;
+        case ASN1_R_DIGEST_AND_KEY_TYPE_NOT_SUPPORTED:
```

Comment:
Worth tossing a test somewhere?

Reply:
Definitely... Will do.

(there's an indirect test in the Android platform but that doesn't really cut it)